libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -180,6 +180,25 @@ void IRGeneratorForStatements::endVisit(VariableDeclarationStatement const& _var
 				declare(m_context.addLocalVariable(*decl));
 }
 
+bool IRGeneratorForStatements::visit(Conditional const& _conditional)
+{
+	_conditional.condition().accept(*this);
+
+	string condition = expressionAsType(_conditional.condition(), *TypeProvider::boolean());
+	auto type = _conditional.annotation().type;
+	IRVariable toAssign(m_context.newYulVariable(), *type);
+	define(toAssign) << m_utils.zeroValueFunction(*type) << "()\n";
+
+	m_code << "switch " << condition << "\n" "case 0 {\n";
+	_conditional.falseExpression().accept(*this);
+	m_code << toAssign.name() << " := " << expressionAsType(_conditional.falseExpression(), *type) << "\n}\n" "default {\n";
```

Comment:
```suggestion
	assign(toAssign, _conditional.falseExpression());
	m_code << "}\n" "default {\n";
```
Same for ``trueExpression()``